### ISSUE TITLE
Accuracy 55% in model-attempt-1 approach

### ISSUE THREAD

**Issue by sanketfirodiya** [humphd/have-fun-with-machine-learning#20]
I have followed the exact steps from the tutorial. Tried with base learning rate 0.01 (default value) and 0.001 (based on the tutorial screenshot), but both didn't get better accuracy. Also it took almost 25 minutes to train.

<img width="1792" alt="screen shot 2018-06-03 at 7 28 16 pm" src="https://user-images.githubusercontent.com/1585037/40895497-bdcaaa80-6764-11e8-9abd-3444baf79426.png">

**Comment by sanketfirodiya** [comment]
cc @humphd @BirkhoffLee would appreciate if you could take a look :)

**Comment by zhongjie526** [comment]
Exactly same issue here

**Comment by humphd** [comment]
Not sure what you're hitting.  Is this retraining the existing models or from scratch?  I'd advise moving on to retrain, since doing the first from-scratch run is not going to produce anything useful regardless (not enough data).

It's possible that something has changed (version issues upstream) with digits or caffe or something since I wrote this, I haven't re-run the steps for a while.

I'm in the process of updating the docs to use TensorFlow at the moment, which I hope to get done this summer.
